akka-stream/src/main/scala/akka/stream/impl/ActorProcessor.scala

```diff
@@ -250,7 +250,8 @@ private[akka] abstract class ActorProcessorImpl(val settings: MaterializerSettin
   protected def onError(e: Throwable): Unit = fail(e)
 
   protected def fail(e: Throwable): Unit = {
-    log.error(e, "failure during processing") // FIXME: escalate to supervisor instead
+    // FIXME: escalate to supervisor
+    log.debug("fail {} due to: {}", self, e.getMessage)
```

Comment:
hmm, maybe we want to protect this debug output with a config option: "stream.debug-log-all-failures" or something like that.